OtripleS.Web.Api/Services/Foundations/Contacts/ContactService.cs

```diff
@@ -50,9 +50,9 @@ public partial class ContactService : IContactService
         TryCatch(async () =>
         {
             ValidateContactId(contactId);
-            Contact contact = await this.storageBroker.SelectContactByIdAsync(contactId);
-            ValidateStorageContact(contact, contactId);
-            return contact;
+            Contact maybeContact = await this.storageBroker.SelectContactByIdAsync(contactId);
+            ValidateStorageContact(maybeContact, contactId);
+            return maybeContact;
```

Comment:
there should be a line between return and previous lines.

Reply:
Yes, you are right. Thanks for letting me know. For this contribution, I was told to not fix everything. So, I only concentrate on change the name.